database/sql/V033__new_table_pre_interactions.sql

```diff
@@ -0,0 +1,10 @@
+-- Creates new table for pre_interactions of orders
+
+CREATE TABLE pre_interactions (
+    order_uid bytea NOT NULL,
+    target_to bytea NOT NULL,
+    value numeric(78,0) NOT NULL,
+    data bytea NOT NULL
+);
+
+CREATE INDEX uid ON pre_interactions USING HASH (order_uid);
```

Comment:
This allows multiple rows per order right? To me, it makes sense to allow a single order to specify multiple pre-interactions.